src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,41 @@
-export const Tabs = () => {};
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+import '../../App.scss';
+import 'bulma/css/bulma.css';
+import '@fortawesome/fontawesome-free/css/all.css';
+
+type Props = {
+  tabs: { id: string; title: string; content: string; }[];
```

Comment:
this can be declared above as:

```
interface Tab {
  id: string,
  title: string,
  content: string,
}
```

and use it here directly:

```
type Props = {
  tabs: Tab[],
  ...etc...
}
```